packages/server/src/commands/export/index.ts

```diff
@@ -99,17 +104,41 @@ export default function exportCommand(command: ServerCommandExport) {
   const dts = generateAppDeclarationFile(app, typeChecker)
   const pheroExecution = generateRPCProxy(app, typeChecker)
 
-  const readFile = (filePath: string): string =>
-    fs.readFileSync(filePath, {
-      encoding: "utf-8",
-    })
+  const lockFile =
+    findLockFileInDir(projectPath) ?? findLockFileForWorkspace(projectPath)
+
+  if (!lockFile) {
+    throw new Error(
+      "No lockfile found in the current directory or for any workspace",
+    )
+  }
 
-  const metaExportFiles: MetaExportFiles = {
+  const metaExportFilesBase: MetaExportFilesBase = {
     "phero-manifest.d.ts": dts,
     "phero-execution.js": pheroExecution.js,
     "phero.js": readFile(path.join(tsOutDir, "phero.js")),
     "package.json": readFile(path.join(projectPath, "package.json")),
-    "package-lock.json": readFile(path.join(projectPath, "package-lock.json")),
+  }
+  let metaExportFiles: MetaExportFiles
+  switch (lockFile[0]) {
```

Comment:
Maybe make the `findLockFileInDir` functions return an object like `{ name: MetaExportLockFileName, path: string}` instead of a tuple. I find `lockFile[0]` not very clear.
Alternatively just unpack the tuple `const [name: lockFileName, path: lockFilePath] = lockFile`.

Reply:
Yeah good point, I was on the fence about that for quite a while. Fixed 👍